README.md

```diff
@@ -3,3 +3,49 @@
 Spring conducted a phenotypic, high content imaging screen on inflammasome activated primary human PBMCs using a library of bioactive small molecules to identify novel inflammasome inhibitors. Using images from these phenotypic datasets, we established a computational method to score phenotypic similarity in order to identify compounds that are likely biologically related.
```

Comment:
Maybe leave out "phenotypic". I have always seen this referred to as HCS or HCI. Phenotypic is more of the intent that such features will come out after the experiment.